boot/assets.go

```diff
@@ -257,11 +257,11 @@ type TrustedAssetsInstallObserver struct {
 
 	blName        string
 	managedAssets []string
-	trustedAssets []string
+	trustedAssets map[string]bool
```

Comment:
I have changed this type to be able to see all the place were we could have expected order to have meaning. This change can be reverted before merge.